components/hero/Hero.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styles from './hero.module.css'
+const Hero = () => {
```

Comment:
For better readability and consistency keep a space between imports and the beginning of the arrow function.